loops.js

```diff
@@ -44,9 +53,16 @@
 // >> 8
 // >> 9
 // ==========================================
-
-
-
+console.log("============Opdracht 3==============================");
+for (let i = 0; i < 10; i++) {
+    if (i > 5) {
+        console.log(`>> ${i}`);
+    } else if (i > 2) {
```

Comment:
Deze code werkt ook prima, echter bij NOVI uitwerking kiezen ze er wel voor om de conditie duidelijker te vermelden (> dan 2  en <6 ) zodat er geen overlap kan plaatsvinden. Maargoed, ik vermoed dat je daarom die >5 conditie eerst hebt vernoemd. Zodat als dat niet het geval is, hij daarna gaat checken of het dan wel groter dan 2 is.